api/main_endpoints/routes/Messages.js

```diff
@@ -1,38 +1,36 @@
-const {
-  UNAUTHORIZED,
-  BAD_REQUEST,
-  SERVER_ERROR,
-  OK
-} = require('../../util/constants').STATUS_CODES;
-const { MAX_AMOUNT_OF_CONNECTIONS } = require('../../util/constants').MESSAGES_API;
+const { UNAUTHORIZED, BAD_REQUEST, SERVER_ERROR, OK } =
+  require('../../util/constants').STATUS_CODES;
+const { MAX_AMOUNT_OF_CONNECTIONS } =
+  require('../../util/constants').MESSAGES_API;
 const express = require('express');
 const router = express.Router();
 const bodyParser = require('body-parser');
 const User = require('../models/User.js');
 const logger = require('../../util/logger');
 const client = require('prom-client');
-const { decodeToken, decodeTokenFromBodyOrQuery } = require('../util/token-functions.js');
+const {
+  decodeToken,
+  decodeTokenFromBodyOrQuery,
+} = require('../util/token-functions.js');
 const { MetricsHandler, register } = require('../../util/metrics.js');
 
-
 router.use(bodyParser.json());
 
-
-
 const clients = {};
 const numberOfConnections = {};
 const lastMessageSent = {};
 
-const writeMessage = ((roomId, message, username) => {
-
+const writeMessage = (roomId, message, username) => {
   const messageObj = {
     timestamp: Date.now(),
     message,
-    username
+    username,
   };
 
   if (clients[roomId]) {
-    clients[roomId].forEach(res => res.write(`data: ${JSON.stringify(messageObj)}\n\n`));
+    clients[roomId].forEach((res) =>
+      res.write(`data: ${JSON.stringify(messageObj)}\n\n`)
+    );
   }
```

Comment:
there are a bunch of random changes in this file. lets disable prettier on vscode, and undo these changes

the only diff in this file should be the logic to remove the mongodb query